roles/provisioner/virsh/tasks/create_vms.yml

```diff
@@ -0,0 +1,78 @@
+---
+- name: set fact for node
+  set_fact:
+      node_data: "{{ item.value }}"
+      node_name: "{{ item.key }}"
+
+- name: import vm disk image(s) for {{ node_name }}
+  get_url:
+      url: "{{ item.value.import_url }}"
```

Comment:
This is another loop over all the disks. So item will point to the disk data. I'm downloading images for all the disks.